exercises/03-is_prime.py

```diff
@@ -1,12 +1,13 @@
 def is_prime(n):
-    for i in range(2, n):
-        if n % i == 0:
-            return False
-    return True
-
+    if n > 1:
+        for i in range(2, int(n/2)+1):
```

Comment:
This is an improvement, but to improve even more you can use the square root instead of `n/2`. For that, you can use:
```
import math
...
math.sqrt(...)
```